frontend/challedu-app/src/App.css

```diff
@@ -8,40 +7,83 @@
   background: url(../src/images/wallpaper.svg);
   background-repeat: repeat;
   background-size: cover;
-  font-family:'Segoe UI';
+  font-family: "Segoe UI";
```

Comment:
You need to add a list of fonts, from the more specific to the more generic one so that it works on every support

Reply:
Something like that?
font-family:courier,arial,helvetica

Reply:
well you also need Segoe UI first I guess ?